<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsFormMain</class>
 <widget class="QMainWindow" name="settingsFormMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>210</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>210</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>210</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Настройки LastVK</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>180</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionSettingsVK">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/buttons/settings/vkontakte_button.png</normaloff>:/buttons/settings/vkontakte_button.png</iconset>
   </property>
   <property name="text">
    <string>ВКонтакте</string>
   </property>
  </action>
  <action name="actionSettingsLastFM">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/buttons/settings/lastfm_button.png</normaloff>:/buttons/settings/lastfm_button.png</iconset>
   </property>
   <property name="text">
    <string>LastFM</string>
   </property>
  </action>
  <action name="actionSettingsMusicLib">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/buttons/settings/musiclib_button.png</normaloff>:/buttons/settings/musiclib_button.png</iconset>
   </property>
   <property name="text">
    <string>Настройки сохранения</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
